### DISCUSSION TITLE
Enforce rate start_minute / end_minute coverage

### DISCUSSION THREAD

**Discussion by zanebclark** [openmobilityfoundation/curb-data-specification#55] [Data Specification]
I'd like to propose that the following requirements be introduced to the Rate / Rule spec: 

>The complete set of rates associated with a given rule must span from start_minutes = 0 or null to end_minutes = rule.max_stay
>There can be no overlap of rates associated with a single policy

If a user were to create the following rate, there would be no rate applicable between minutes 1-9 and after minute 15: 
- `per_hour` = 25
- `start_minutes` = 10 
- `end_minutes` = 15 

If the intent was to allow for free parking up until 10 minutes, the user would initially need to define the following:
- `per_hour` = 0
- `start_minutes` = 0
- `end_minutes` = 9

Furthermore, if the intent was to allow for free parking after minute 15, the user would also need to define the following:
- `per_hour` = 0
- `start_minutes` = 16
- `end_minutes` = null

This would achieve the following requirement: 
>The complete set of rates associated with a given rule must span from start_minutes = 0 or null to end_minutes = rule.max_stay

**Comment by schnuerle** [comment]
I see your point. Are you suggesting just adding some clarifying language like you suggest, to make this requirement clear?

**Reply by zanebclark** [reply]
Sounds like a plan!

**Reply by schnuerle** [reply]
Feel free to make a PR and add the language where you are thinking it should go. Make it against the feature-release-work-1 branch.  It makes you a contributor too! Or I will make the edit sometime soon.

**Reply by zanebclark** [reply]
Filed https://github.com/openmobilityfoundation/curb-data-specification/pull/57
